system_files/shared/etc/geoclue/conf.d/99-beacondb.conf

```diff
@@ -0,0 +1,3 @@
+[wifi]
+enable=true
+url=https://api.beacondb.net/v1/geolocate
```

Comment:
Is this a valid URL?

```
curl https://api.beacondb.net/v1/geolocate -i
HTTP/2 404 
server: nginx
date: Wed, 12 Feb 2025 23:50:58 GMT
content-length: 0
```

Reply:
It should be, this entire thing is from the Fedora magazine.

Reply:
It works using a POST request.  GETs return a 404